app/schemas/de.tum.in.tumcampusapp.database.TcaDb/6.json

```diff
@@ -2,18 +2,12 @@
   "formatVersion": 1,
   "database": {
     "version": 6,
-    "identityHash": "f743f786b0866afd26a4cc1c1fafc8d1",
+    "identityHash": "07b6e7b518ae4df546549d9a64c87963",
```

Comment:
As far as I know, existing migrations should not be changed. Could you have another look into why this was changed?